File: AndroidApp/app/src/main/java/org/duckdns/sunga/rw5noti/home/MessageViewModel.java
package org.duckdns.sunga.rw5noti.home;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class MessageViewModel extends ViewModel {

    private final MutableLiveData<String> _text = new MutableLiveData<>();

    public MessageViewModel() {
        _text.setValue("This is dashboard Fragment");
    }

    public LiveData<String> getText() {
        return _text;
    }
}

File: Android3/app/src/main/java/com/xiaomi/xms/wearable/demo/ui/message/MessageViewModel.kt
package com.xiaomi.xms.wearable.demo.ui.message

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class MessageViewModel : ViewModel() {

    private val _text = MutableLiveData<String>().apply {
        value = "This is dashboard Fragment"
    }
    val text: LiveData<String> = _text
}